integration-test/src/test/scala/io/phdata/clients/RoleClientIntegrationSpec.scala

```diff
@@ -0,0 +1,210 @@
+package io.phdata.clients
+
+import java.util.UUID
+
+import cats.effect.IO
+import io.phdata.itest.fixtures.{HiveTest, IntegrationTest, _}
+import io.phdata.models.{Manager, ReadOnly, ReadWrite}
+import io.phdata.services.UGILoginContextProvider
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import cats.implicits._
+
+class RoleClientIntegrationSpec extends FlatSpec with Matchers with HiveTest with IntegrationTest with BeforeAndAfterAll{
+
+  val hiveClient = new HiveClientImpl[IO](new UGILoginContextProvider(itestConfig), hiveTransactor)
+  val roleClient = new RoleClientImpl[IO](hiveTransactor, null, new UGILoginContextProvider(itestConfig))
+
+  val role = s"test_role_${UUID.randomUUID().toString.take(8)}"
+  val group = "integration_test_group"
+  val testDatabase = "sentry_client_integration_test"
+  val location = "integration_test"
+
+  override def beforeAll() {
+    val init = for {
+      existingDBs <- hiveClient.showDatabases()
+      _ <- if (!existingDBs.contains(testDatabase)) {
+        hiveClient.createDatabase(testDatabase, "/tmp", "a comment", Map("pii_data" -> "true", "pci_data" -> "false"))
+      } else {
+        ().pure[IO]
+      }
+    } yield ()
+
+    init.unsafeRunSync()
+  }
+
+  override def afterAll() {
+    val cleanUp = for {
+     roles <- roleClient.roles
+     _ <- roles.filter(_.startsWith("test_role_")).map(testRole => roleClient.dropRole(testRole)).pure[IO]
+    } yield ()
+
+    cleanUp.unsafeRunSync()
+  }
+
+  behavior of "Sentry Client"
+
+  it should "list roles" in {
+    val testRoles = List(
+      s"test_role_${UUID.randomUUID().toString.take(8)}",
```

Comment:
do all roles in this class start with`test_role`? It's good if they do, makes it easier to clean up if something goes wrong. Maybe better, make it `arch_test_role`, then all the objects we create start with `arch_test` and we can easily recognize them